### DISCUSSION TITLE
Block Height as warning

### DISCUSSION THREAD

**Discussion by sl00k** [smartcontractkit/full-blockchain-solidity-course-py#385] [Q&A] [answered]
Hi everyone,

why does brownie give me a warning here. why does it expect a block height of 0, like in the video
(and why is mine going up while in the video it always starts at 0 again)

When I push the contract on a live blockchain the block height won't be zero as well, so I dont know why it is a warning here
![image](https://user-images.githubusercontent.com/33859801/142223122-62b71805-a286-4fcf-98a0-60344e13e009.png)

**Comment by cromewar** [comment, marked as answer]
Hello @sl00k this warning was already discussed on official Brownie repo, please refer to [this](https://github.com/eth-brownie/brownie/issues/1075) link for a explanation. Anyways you have nothing to worry about ;).
